jobs/aws/eks-distro-build-tooling/alertmanager-postsubmits.yaml

```diff
@@ -47,12 +47,15 @@ postsubmits:
         command:
         - bash
         - -c
+        - env:
```

Comment:
env doesn't come in the `command` section. There's a separate field in the container spec. Refer to this job in your diff https://github.com/aws/eks-distro-prow-jobs/pull/405/files#diff-a3e0d8ce9a04e981c1131e9b061ea4fdcf0f93246e50e1f9a97e2fc2f78b13e5 to see where the env section goes. There can be only one env section in a container spec. The templater handles envvars already, if you define a new env section in the minimal config, that should do it.